packages/docs/src/routes/qwikcity/api/index.mdx

```diff
@@ -116,10 +117,42 @@ The above code would generate:
 
 ```html
 <h1>SKU</h1>
-<p>pathname: /sku/1234</p>
+<p>pathname: /sku/1234/</p>
 <p>skuId: 1234</p>
 ```
 
+You can also use `...` syntax to indicate deeply nested child routes. For example:
```

Comment:
Yes, sort of. The page you mentioned never showed an example of using `params` that represent deeply nested routes (that are located within folder that starts with `[...`). I thought that the API Reference page should cover the details, though I'm not sure did the right job. Maybe propose something? 🙂

BTW, I somehow read that the type of `params` filed was `Readonly<Record<string, any>>` (instead of `Readonly<Record<string, string>>`) which got me confused (by my mistake) that `params` properties can have any value. This is why I wanted to document this so that other people don't get confused as well, but they probably won't since `params.anyParamHere` will always be `readonly string`.